### ISSUE TITLE
Google closed this

### ISSUE THREAD

**Issue by fenixryan** [tuomaskivioja/3-Simple-Python-Projects#3]
now receiving KeyError: 'streamingData' trying to run script

**Comment by azeemprogrammer** [comment]
Yes, it is true. For personal Purpose Google allow you to Use YouTube Data. But I have modified this old script now it working Perfectly. As for why the old code was not working, the error you encountered,` IndexError: list index` out of range, was due to the fact that you were trying to access `argv[1]` to get the YouTube URL from the command-line arguments
